opsdroid/conftest.py

```diff
@@ -1,47 +1,7 @@
 """Pytest config for all opsdroid tests."""
```

Comment:
```suggestion
"""Pytest config for all opsdroid tests.
This file includes definitions for any private (opsdroid-use-only) fixtures.
All public fixtures should be defined in opsdroid.testing.fixtures so that sub-projects can reuse them.
"""
```